pravega_provisioner/cluster_provisioner.py

```diff
@@ -154,32 +181,50 @@ def resource_based_provisioning(vms, vm_cpus, vm_ram_gb, vm_local_drives, zookee
     # Get the resources used for the last possible allocation
     the_cluster = can_allocate_services_on_nodes(vms, vm_cpus, vm_ram_gb, vm_local_drives, zookeeper_servers,
                                                  bookkeeper_servers, segment_stores, controllers)[1]
-    print("Allocation of pods on nodes: ", the_cluster)
+    if verbose:
```

Comment:
Please, remove all the `if verbose`, this makes the code more complex without need. The script may print things anyway, I don't see why it should not (it can be used anyway from another program).

Reply:
done. Also I rounded the decimals to 2.
- Write throughput:  100 (MBps) -> buffering time:  6615.04  seconds
- Write throughput:  300 (MBps) -> buffering time:  2205.01  seconds
- Write throughput:  500 (MBps) -> buffering time:  1323.01  seconds
- Write throughput:  700 (MBps) -> buffering time:  945.01  seconds
- Write throughput:  900 (MBps) -> buffering time:  735.0  seconds
old output:
- Write throughput:  100 (MBps) -> buffering time:  6615.04  seconds
- Write throughput:  300 (MBps) -> buffering time:  2205.0133333333333  seconds
- Write throughput:  500 (MBps) -> buffering time:  1323.008  seconds
- Write throughput:  700 (MBps) -> buffering time:  945.0057142857142  seconds
- Write throughput:  900 (MBps) -> buffering time:  735.0044444444444  seconds